### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by Edinanorge**
project almost done

### CODE REVIEW

File: home.html [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+              <a href="#" class="card-link text-white btn" aria-label="like post"><i class="bi bi-heart-fill"></i></a>
+              <a href="#" class="card-link text-white btn" aria-label="send message"><i class="bi bi-chat-fill"></i></a>
+              <a href="#" class="card-link text-white btn" aria-label="share"><i class="bi bi-share-fill"></i></a>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- flex 3 item d-none on mobile -->
+      <div class="d-none d-lg-block mx-lg-5">
```
Comment (Fermain) [inline home.html]: Hiding content is not a responsive strategy, please add responsive classes and/or row-col in order to accommodate  smaller screens. Mobile users deserve the same experience as desktop

File: home.html [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+            <i class="bi bi-camera"></i>
+          </button>
+          <button class="btn btn-primary border-primary ms-3" aria-label="send new post">
+            <i class="bi bi-send-fill"></i>
+          </button>
+        </div>
+
+        <!-- feeds -->
+        <div class="row feed">
+          <div class="card bg-gray col-12 mx-auto">
```
Comment (Fermain) [inline home.html]: I suggest using an extra level of div here:

```html
<div class="row feed">
  <div class="col-12">
    <div class="card bg-gray mx-auto">
```

File: profile.html [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <main class="container profil">
+      <div class="d-lg-flex justify-content-evenly">
+        <!-- flex item 1 -->
+        <!-- profil img, details, form  -->
+        <div class="container text-center m-lg-5 px-lg-5">
```
Comment (Fermain) [inline profile.html]: You shouldn't have a `.container` nested inside another `.container`.

File: profile.html [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+            <p class="fs-4 text-white mb-0">Jessica Jhones</p>
+            <div class="col-lg-6 mx-auto">
+              <p class="fs-6 mb-4 text-secondary">London, United Kingdom</p>
+              <div class="d-grid gap-2 d-sm-flex justify-content-sm-center">
+                <button type="button" class="btn btn-outline-light btn-sm px-4 me-sm-3">Edit profile</button>
+                <a href="/index.html" type="button" class="btn btn-outline-light btn-sm px-4">Log out</a>
+              </div>
+            </div>
+          </div>
+          <div class="d-flex flex-row add-post">
```
Comment (Fermain) [inline profile.html]:
```suggestion
          <form class="d-flex flex-row add-post">
```

File: profile.html [hunk 5]
```diff
@@ -0,0 +1,251 @@
...
+                class="rounded-circle m-1 border border-secondary border-2 d-none d-md-flex"
+              />
+              <p class="m-0 ps-md-2 d-none d-lg-block">Nora</p>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <!-- latest post-->
+      <div class="container">
```
Comment (Fermain) [inline profile.html]: Nested container again

File: profile.html [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+        <p class="fs-4 text-muted mt-3">Latest posts</p>
+        <div class="row row-cols-1 row-cols-sm-2 row-cols-md-3 g-1 gallery">
+          <div class="col">
+            <div class="card shadow-sm">
+              <img src="/src/images/feed1.jpg" alt="" />
```
Comment (Fermain) [inline profile.html]: Empty alt tag no classes? Suggestion `.img-fluid`

File: src/scss/styles.scss [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+  @include image-size(5rem, 5rem, cover);
+}
+// create post
+.add-post {
+  input {
+    padding: 0.7rem;
+  }
+
+  .btn-emoji {
+    margin-right: -2.7rem;
```
Comment (Fermain) [inline src/scss/styles.scss]: This is unusual, could you explain your choice here?

File: src/scss/styles.scss [hunk 8]
```diff
@@ -0,0 +1,112 @@
+@import "variables.scss";
+
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
+// general
+.fs-6 {
```
Comment (Fermain) [inline src/scss/styles.scss]: By extending rather than adding to bootstrap you can achieve these results. Please see loom video for demonstration.

**Review by Fermain — CHANGES_REQUESTED** [review]
Some minor changes here, but overall a very good looking project and varied use of SASS with bootstrap to produce an original theme.

**Comment by Fermain** [comment]
https://www.loom.com/share/f9be604237424030a8894acca6ac365a

File: home.html (hunk 2 above)
Comment (Edinanorge) [reply home.html]: Fixed it.

File: profile.html (hunk 4 above)
Comment (Edinanorge) [reply profile.html]: Fixed.

File: profile.html (hunk 5 above)
Comment (Edinanorge) [reply profile.html]: Corrected.

File: src/scss/styles.scss (hunk 8 above)
Comment (Edinanorge) [reply src/scss/styles.scss]: Done. :)

File: profile.html (hunk 3 above)
Comment (Edinanorge) [reply profile.html]: Fixed it.

File: src/scss/styles.scss (hunk 7 above)
Comment (Edinanorge) [reply src/scss/styles.scss]: This was just a quick and dirty solution to visually put some buttons inside an input tag. But I still looking for a better way to do that. :)

File: profile.html (hunk 6 above)
Comment (Edinanorge) [reply profile.html]: Fixed.

File: src/scss/styles.scss (hunk 7 above)
Comment (Edinanorge) [reply src/scss/styles.scss]: Solved whit custom code - positioning.

File: home.html (hunk 1 above)
Comment (Edinanorge) [reply home.html]: Fixed.

**Pull request merged**